packages/@okta/vuepress-site/docs/guides/submit-app/overview/index.md

```diff
@@ -1,22 +1,34 @@
 ---
-title: Publishing an app integration with Okta Integration Network
+title: Publish an app integration with Okta Integration Network
 ---
 
-This guide outlines the process by which independent software vendors (ISVs), Okta customers, or IT system integrators can submit an app for review by Okta with the goal of publishing it to the [Okta Integration Network](https://www.okta.com/integrations/). Integrating your application with the Okta Integration Network is completely cost free.
+If you are an independent software vendor (ISV), Okta customer, or IT system integrator who wants to add their app integration to the [Okta Integration Network](https://www.okta.com/integrations/), read this guide for instructions on how to submit your app. Integrating your application with the Okta Integration Network is completely cost free.
```

Comment:
To provide general context, would it be worthwhile to add here some possible reasons why you might want to add an app to the OIN, why it's good for you to have your app in the OIN?